plugins/destination/file/client/migrate.go

```diff
@@ -0,0 +1,12 @@
+package client
+
+import (
+	"context"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func (*Client) Migrate(ctx context.Context, tables schema.Tables) error {
+	// migrate is not needed in migrate mode
```

Comment:
What is the behavior intended to be if the columns for a CSV file change between versions, and a sync is run? Do we always overwrite the file? (In which case `append` is a bit confusing) Or will it append values in the incorrect positions or with additional columns not matched by the headers?

Reply:
we always overwrite. also we always write to `.format.uuid` unless `NoRotate` is specified (need to document that indeed). but yeah it is up to the user. implementing migration over files is quite hard

Reply:
Maybe we should use sync time instead of uuid, so the files are at least sorted in increasing order?

Reply:
we need uuid because if those are uploaded to remote storage it can cause issues and uuid is the only way to avoid collision if it is running in distributed mode.

Reply:
You could use a `ksuid`. Segment has a good golang library: https://github.com/segmentio/ksuid

Reply:
Nice, I was also going to suggest a Snowflake ID that @candiduslynx mentioned on another thread the other day: https://en.wikipedia.org/wiki/Snowflake_ID. Seems like an ideal fit for this case

Reply:
yes, we can use one of those time based IDs but for now I don't think it's yet needed?

Reply:
I think there's a version of UUID that is sortable https://github.com/oklog/ulid (or we can just contact time+UUID)

Reply:
ksuid is very nice (I've used it lots before) the only downside is they look a bit weird.